apps/web/pages/success.tsx

```diff
@@ -164,17 +164,20 @@ export default function Success(props: SuccessProps) {
 
   const attendeeName = typeof name === "string" ? name : "Nameless";
 
+  console.log(props.eventType.locations);
+
   const eventNameObject = {
     attendeeName,
     eventType: props.eventType.title,
     eventName: (props.dynamicEventName as string) || props.eventType.eventName,
     host: props.profile.name || "Nameless",
+    location: props.eventType.locations[0].type || "",
     t,
   };
   const metadata = props.eventType?.metadata as { giphyThankYouPage: string };
   const giphyImage = metadata?.giphyThankYouPage;
 
-  const eventName = getEventName(eventNameObject);
+  const eventName = getEventName(eventNameObject, true);
```

Comment:
On the success page we want the dynamic name to be from the perspective of the attendee

Reply:
The success page is also the booking detail view of the host. That means that the host will see its own name then

Reply:
@CarinaWolli I think when user host reach success page he still wanna see it as an Attendee. This is due no user Host will self book it self.

Reply:
@alannnc  I wasn't talking about the success page where we redirect to when someone successfully books a meeting. We also redirect to the same success page once a host clicks on an upcoming booking to see the booking details. I think in that case the host wants to see the attendee's name and not it's own

Reply:
This should fix it:
```suggestion
  const eventName = getEventName(eventNameObject, listingStatus ? false : true);
```